src/fontra/client/web-components/ui-form.js

```diff
@@ -1,23 +1,22 @@
 import { QueueIterator } from "../core/queue-iterator.js";
 import { hyphenatedToCamelCase, round } from "../core/utils.js";
-import { css } from "../third-party/lit.js";
+import { css, unsafeCSS } from "../third-party/lit.js";
```

Comment:
There is no need to use `css` or `unsafeCSS` with `SimpleElement`. We should not import lit unless we're actually using lit.

Or do you use `css` to trigger syntax highlighting in your editor?